.github/workflows/ansible-test.yml

```diff
@@ -75,15 +75,10 @@ jobs:
       # The docker container has all the pinned dependencies that are
       # required and all Python versions Ansible supports.
       - name: Perform sanity testing
-        uses: ansible-community/ansible-test-gh-action@release/v1
```

Comment:
@samdoran `release/v1` actually tracks the very latest release. Using specific tags, you'll have to remember to update them manually, and there's a greater chance you'll use outdated action versions.

Reply:
I was wondering about that but couldn't find documentation explaining how `@release` works.

Reply:
Ah, it's just a Git branch. I use this pattern in all my actions to be explicit that it's a moving pointer as opposed to the immutable tags I make for each release.

Reply:
Oh! That makes total sense. I kept looking for that to be a special keyword of some sort. I was just overthinking it. 🤦